src/functions/unreferencedReusableObject.ts

```diff
@@ -0,0 +1,28 @@
+import { IFunction } from '../types';
+
+export type OasReusableObjectsLocation = '#/definitions' | '#/components/schemas';
+
+export const unreferencedReusableObject: IFunction<{ reusableObjectsLocation: OasReusableObjectsLocation }> = (
```

Comment:
Each core function is documented, so this one should be no different.
I can do it, though, no worries. :)
Just leaving a note so that we don't forget about it.

Reply:
Done.